src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4062,7 +4062,7 @@ namespace ConvectionCoefficients {
         using namespace DataZoneEquipment;
 
         int ZoneNode = Zone(ZoneNum).SystemZoneNodeNumber;
-        int thisZoneInletNode;
+        int thisZoneInletNode = 0;
```

Comment:
This is a little fix I added to try to help the convection unit tests pass.  Unrelated to this work, move along...